com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -1398,10 +1399,11 @@ RTHandle Allocator(string id, int frameIndex, RTHandleSystem rtHandleSystem)
 
         void GrabVelocityMagnitudeHistoryTextures(HDCamera camera, out RTHandle previous, out RTHandle next)
         {
+            Vector2 scale = new Vector2(camera.postProcessScreenSize.x * camera.screenSize.z, camera.postProcessScreenSize.y * camera.screenSize.w);
```

Comment:
Does this create GC Alloc? It is captured in the lambda, I solved this in the TAAU PR with a different way to handle allocators.

Reply:
This function is not executed inside a lambda (In this version of the code), so there is no GC alloc. I guess you have moved this in your PR? We will certainly have to merge a few things in the end.
(I'm also runnign a yamato job as we speak to check for any errors and gc allocs)

Reply:
The parameter scale is captured by the allocator lambda https://github.com/Unity-Technologies/Graphics/pull/5189/files#diff-617165acfec9a5b4c1b26809291f30938d7a152b8e9fec0e5ae2c661078201dbR1403

Reply:
Yes,  you are right. Interestingly our tests did not catch any GC allocs, but maybe the guards are not ON when this is executed.

Since you have solved this in the TAAU PR, I guess I should rebase this PR on top of TAAU and also fix any merge conflicts.

Reply:
One thing we can do is that we can close this PR and I do the change on top of TAAU _or_ we wait for TAAU to land and you add this on top of the rest.

Reply:
Ok, for now I have added [Hold] in the title to wait for TAAU, but if you have time to apply this change in your PR, poke me and I will close this one. Otherwise I will merge after your PR has landed.